package/gluon-static-ip/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-static-ip
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-static-ip
+  TITLE:=Static IP assignment and configuration for gluon
+  DEPENDS:=+gluon-core +luci-lib-ip
```

Comment:
we'd need luci-lib-ip from the luci repo. in the fork currently the entire luci repo is simply added.